diff --git a/rofi-bluetooth b/rofi-bluetooth
index 16ca443..0603208 100755
--- a/rofi-bluetooth
+++ b/rofi-bluetooth
@@ -186,7 +186,7 @@ print_status() {
 
         paired_devices_cmd="devices Paired"
         # Check if an outdated version of bluetoothctl is used to preserve backwards compatibility
-        if (( $(echo "$(bluetoothctl version | cut -d ' ' -f 2) < 5.65" | bc -l) )); then
+        if (( $(echo "$(bluetoothctl version | head -n 1 | cut -d ' ' -f 2) < 5.65" | bc -l) )); then
             paired_devices_cmd="paired-devices"
         fi
 
